content/markdown/concepts/highlight/highlight.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'highlight'
+Description: 'Highlights are when two pairs of equal signs enclose a piece of text, resulting a yellow background encompassing said text.'
+Subjects:
+  - 'Web Development'
+  - 'Developer Tools'
+Tags:
+  - 'Documentation'
+CatalogContent:
+  - 'learn-html'
+  - 'path/front-end-engineer-career-path'
+---
+
+In Markdown, **highlight** is where parts of a text appears in a yellow background after being inserted between two pairs of equal signs (`==`).
+
+> **Note:** This feature is not offered in all Markdown processors so consider reading through the instruction guide before completing.
+
+## Syntax
+
+```md
+==Text==
+```
+
+## Example
+
+The range of numbers in the below baking tip is highlighted so that users understand the amount of cookies they will have based on the amount of dough made.
+
+```md
+Depending on how much cookie dough you eat, you’ll have about ==20-22== cookies from this recipe!
+```
```

Comment:
Can you make a block quote after this code block to show what it will look like as markdown?

Like this:

> Depending on how much cookie dough you eat, you’ll have about ==20-22== cookies from this recipe!

Preface as something along the lines of, "This will appear as:"

Reply:
@CBID2 I added this to use the <mark> tag and will make a note that this is a good alt